Charging_CurrentTest.cpp

```diff
@@ -0,0 +1,13 @@
+#define CATCH_CONFIG_MAIN // This tells Catch to provide a main() - only do this in one cpp file
+
+#include "catch.hpp"
+#include "Charging_Current.h"
+using namespace std;
+
+TEST_CASE("Analysing Multiple Samples")
+{
+  generateRangeAndPrintOutput({4, 5, 10});
```

Comment:
Test cases need to verify the output. Verification is missing.

Reply:
Fixed